<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envio de incidencias</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enviar incidencias</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Created by lleon</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Path de los archivos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblarchivos">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnArcvhivos">
       <property name="text">
        <string>Seleccionar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>108</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Texto del correo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnviar">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtcorreo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuContactos">
    <property name="title">
     <string>Contactos</string>
    </property>
    <addaction name="actionAgregar"/>
    <addaction name="actionListar"/>
    <addaction name="actionEliminar"/>
   </widget>
   <widget class="QMenu" name="menuSalir">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionSalir"/>
    <addaction name="actionConfiguracion"/>
   </widget>
   <addaction name="menuContactos"/>
   <addaction name="menuSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAgregar">
   <property name="text">
    <string>Agregar</string>
   </property>
  </action>
  <action name="actionListar">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionConfiguracion">
   <property name="text">
    <string>Configuracion</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
